packages/definitions-parser/allowedPackageJsonDependencies.txt

```diff
@@ -236,3 +236,4 @@ xmlbuilder
 xpath
 zipkin
 zipkin-transport-http
+canvas
```

Comment:
Can you insert this in alphabetical order please? Also, please link to the DefinitelyTyped PR that needs this.

Reply:
Fixed the order. Request is for this PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/49209